### PR TITLE
Release 1.1.0 articles

### PR DESCRIPTION
**Pull request by stefonarch**
* For some real mysterious reason obconf-qt release post isn't shown in March but in April, wasn't able to track this down.
* Fixed also a typo in older notificationd articles.

### CODE REVIEW

File: _posts/2022-04-15-lxqt-1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * The new component `xdg-desktop-portal-lxqt` is added to LXQt, as a backend implementation for `xdg-desktop-portal`. When it is installed, some non-Qt apps like Firefox can be set up to use LXQt file dialog.
+ * LXQt themes are improved. A new theme and several wallpapers are added. Also, matching Qt palettes are added for a fairly uniform look and feel with Qt widget styles like Fusion. They can be found and applied in LXQt Appearance Configuration → Widget Style → Qt Palette.
+ * The bookmark functionality is [improved a lot](https://lxqt-project.org/blog/2022/03/15/qterminal-bookmarks-reloaded/) in QTerminal and several (old) issues in the drop-down mode are fixed.
+ * In LXQt Penel, the legacy tray icons are now shown inside Status Notifier when the System Tray plugin is enabled. That removes the problem that the legacy System Tray had with auto-hiding panels.
```
Comment (tsujan) [inline _posts/2022-04-15-lxqt-1.1.0.md:19]: Penel → Panel. The typo is inherited from my text.

**Comment by tsujan** [comment]
It seems OK to me. If there's a problem that we can't see, it shouldn't be important.

Thanks for the hard work!

**Comment by stefonarch** [comment]
Well, the harder work was yours :)
If there are errors we'll fix them ... but after Easter...

**Comment by tsujan** [comment]
> Well, the harder work was yours

Mine doesn't count without yours.

**Pull request merged**